solcx/install.py

```diff
@@ -50,7 +53,7 @@ def _get_platform():
     if sys.platform in ("darwin", "win32"):
         return sys.platform
     raise KeyError(
-        "Unknown platform: '{}' - py-solc-x supports" " Linux, OSX and Windows".format(sys.platform)
+        f"Unknown platform: '{sys.platform}' - py-solc-x supports Linux, OSX" f" and Windows"
```

Comment:
```suggestion
        f"Unknown platform: '{sys.platform}' - py-solc-x supports Linux, OSX and Windows"
```

No need for two strings here.